components/hint-message/hint-message.jsx

```diff
@@ -11,9 +11,8 @@ class HintMessage extends React.Component {
 
   render() {
     return (
-      <div className="hint-message text-center">
-        <h2 className="icon mb-3">{ this.props.iconComponent }</h2>
-        <h2>{ this.props.header }</h2>
+      <div className="pt-4 pt-lg-5 text-center">
```

Comment:
Not a huge issue here as it's just spacing but we should get into a habit of mentioning all the elements affected to design reviewers if we changed a shared component code. This way we don't accidentally introduce design changes we didn't want.

---

`pt-4` (1.5rem top padding) and `pt-lg-5` (3rem top padding for larger screen) don't correspond to the old spacing we set in `hint-message.scss` (3rem / 5rem). As this `hint-message.jsx` component is also used outside of the profile/favs tabs we should get Nat to design review all the `<HintMessage>` occurrences on the site to make sure the new spacing look right on other pages as well.

Reply:
good point, i'll update the initial comment and tag Nat so she can review the other instances.